### PR TITLE
updated mapbox npm install version to install latest patch

### PR DESCRIPTION
**Pull request by barryslice**
### CATEGORY

Choose one

- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Changed mapbox-gl version in package.json from ^0.53.0 to ^0.53.x
npm install mapbox-gl@^0.53.0 doesn't install the latest patch 0.53.1
Version 0.53.1 is required for mapbox token url whitelisting.
See: https://docs.mapbox.com/help/account/tokens/#requirements-and-limitations

### CODE REVIEW

File: superset/assets/package.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
     "jquery": "3.1.1",
     "json-bigint": "^0.3.0",
     "lodash": "^4.17.11",
-    "mapbox-gl": "^0.53.0",
+    "mapbox-gl": "^0.53.x",
```
Comment (etr2460) [inline superset/assets/package.json]: I think it's preferred to just pin directly to `0.53.1`

Comment (barryslice) [reply superset/assets/package.json]: Done

**Comment by etr2460** [comment]
Looks like the lockfile didn't get updated properly. I'd recommend rechecking out master, making the change in `package.json` then running `npm install` and committing all changes

**Comment by barryslice** [comment]
@etr2460 any idea why this causes ``` - /home/travis/build/apache/incubator-superset/superset/assets/cypress/screenshots/explore/control.test.js/Advanced analytics -- Create custom time compare (failed).png (1280x720)``` to fail?

**Comment by mistercrunch** [comment]
deck.gl was migrated here:
https://github.com/apache-superset/superset-ui-plugins-deckgl 

Same PR could be applied there

**Comment by mistercrunch** [comment]
https://github.com/apache-superset/superset-ui-plugins-deckgl/pull/4

**Pull request closed without merging**